server/base/src/main/java/org/apache/accumulo/server/util/MetadataTableUtil.java

```diff
@@ -894,12 +894,19 @@ public static void removeBulkLoadEntries(Connector conn, String tableId, long ti
         BatchWriter bw = conn.createBatchWriter(MetadataTable.NAME, new BatchWriterConfig())) {
       mscanner.setRange(new KeyExtent(tableId, null, null).toMetadataRange());
       mscanner.fetchColumnFamily(TabletsSection.BulkFileColumnFamily.NAME);
+      boolean shouldTrace = log.isTraceEnabled();
+      String tidString = Long.toString(tid);
       for (Entry<Key,Value> entry : mscanner) {
-        log.debug("Looking at entry " + entry + " with tid " + tid);
-        if (Long.parseLong(entry.getValue().toString()) == tid) {
-          log.debug("deleting entry " + entry);
-          Mutation m = new Mutation(entry.getKey().getRow());
-          m.putDelete(entry.getKey().getColumnFamily(), entry.getKey().getColumnQualifier());
+        if (shouldTrace) {
+          log.trace("Looking at entry " + entry + " with tid " + tidString);
```

Comment:
That's a pretty common paradigm that's used in SLF4J. what do you mean by "Achieves the same result?" It should achieve the same result. If it does not then there is an issue with SLF4J, at which point I would agree with you. If it does achieve the same result and the conditional is unnecessary, then why does it exist?